exercises/bob/Cargo.lock

```diff
@@ -1,4 +0,0 @@
-[root]
```

Comment:
Or maybe just all Cargo.lock for exercises that do not declare any dependencies (all exercises except gigasecond and robot-name)???